src/olcs/core.js

```diff
@@ -631,3 +636,16 @@ olcs.core.normalizeView = function(view, angle = 0) {
   view.setRotation(angle);
   view.setResolution(view.constrainResolution(resolution));
 };
+
+/**
+ * Check if the given projection is managed by Cesium (WGS84 or Mercator Spheric)
+ *
+ * @param {ol.proj.Projection} projection Projection to check.
+ * @returns {boolean} Wether it's managed by Cesium.
+ */
+olcs.core.isCesiumProjection = function(projection) {
+  const is3857 = projection === ol.proj.get('EPSG:3857');
+  const is4326 = projection === ol.proj.get('EPSG:4326');
+  return !!(is3857 || is4326);
```

Comment:
is3857 and is4326 are booleans, right? why do you need the parenthesis and !!?

Reply:
Sorry, was not focused.. Thinks it's the final one.